packages/promotion/src/migrations/Migration20240122084316.ts

```diff
@@ -59,7 +59,11 @@ export class Migration20240122070028 extends Migration {
     )
 
     this.addSql(
-      'create table "application_method_promotion_rule" ("application_method_id" text not null, "promotion_rule_id" text not null, constraint "application_method_promotion_rule_pkey" primary key ("application_method_id", "promotion_rule_id"));'
+      'create table "application_method_target_rules" ("application_method_id" text not null, "promotion_rule_id" text not null, constraint "application_method_target_rules_pkey" primary key ("application_method_id", "promotion_rule_id"));'
```

Comment:
Yes, good shout. I've added it to my pre-release tasks in linear. The reason i'm not doing it now is because we will still destroy existing migration files and create new ones a few more times.